src/algorithms/dqns/common.jl

```diff
@@ -35,3 +35,34 @@ function RLBase.update!(
     push!(trajectory[:terminal], is_terminated(env))
     push!(trajectory[:priority], p.learner.default_priority)
 end
+
+#####
+# some common components for Dueling network
+# Dueling network automatically produces separate estimates of the state value function network and advantage function network. The expected output size of val is 1, and adv is the size of the action space.
+#####
```

Comment:
The comment here won't be shown in the docs. Please replace it with:

```
"""
...
"""
```
And move it right above the line of `struct DuelingNetwork`.